src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java

```diff
@@ -0,0 +1,111 @@
+package edu.cmu.cs.cs214.rec10.plugin;
+
+import edu.cmu.cs.cs214.rec10.framework.core.GameFramework;
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+import edu.cmu.cs.cs214.rec10.framework.core.Player;
+
+import java.util.*;
+
+public class MoneyDoorPlugin implements GamePlugin {
+
+    private int[] arr = {-3, -2, -1, 0, 1, 2, 3, 4, 5, 6};
+    private int ONE = 1;
```

Comment:
bad variable naming, indicate what this will be used for

Reply:
pretty descriptive to me ONE == 1